src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,18 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Car get(Long id);
```

Comment:
It will be better you return Optional here
```suggestion
   Optional<Car> get(Long id);
```

Reply:
But why? In the task said, that i need to return Car, not Optional<Car>